pkg/backends/all/all.go

```diff
@@ -0,0 +1,10 @@
+package all
```

Comment:
Why no move this imports to the main.go?

Reply:
Because then I would have to repeat all imports on another packages, see this commit: https://github.com/hellofresh/kangal/pull/75/commits/fa91f60b1caf5f014e18a7d88062c5bfdba37724

Reply:
This all package feel like a hack to me since it break the concept of the folders within backends at the moment. We have no backend/system called `all` that we support. 
Also those `packages` you link to are tests which I would argue should not break if another backend is added to the system which in this case may happen.

I possible solution would be to use an `internal` package to register all backends or to split the backends package into backends and backend.
Where backend would contains the Interfaces and backends the import for all backends in it's subdirs.

Reply:
what do you think now?

Reply:
I'm wondering if naming the package after the pattern `registry` is a good idea. What happens if you want to add another registry to the project? 
Also when I look at `registry.New` for example is unclear to me what the registry is for. Maybe it's a generic registry?

Reply:
I can improve the `registry.New` to be more clear or I can move both, `registry` and `backends` into a `backend` package, but to be honest I don't like, I prefer the first option